packages/block-library/src/code/index.js

```diff
@@ -23,6 +23,13 @@ export const settings = {
 	supports: {
 		html: false,
 	},
+	attributes: {
```

Comment:
As part of the Block Registration RFC (#13693), attributes definitions for core blocks have been moved to `block.json` within the block directory (see #14770). Updates should be made there instead.

Reply:
Ok, thanks! I'll fix this tomorrow first thing in the morning.